### PR TITLE
[OSDOCS-9310]:Openshift Logging error in the ROSA documentation

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.14+
Issue:

https://issues.redhat.com/browse/OSDOCS-9310
Link to docs preview:

https://70675--ocpdocs-pr.netlify.app/openshift-rosa/latest/logging/cluster-logging-deploying#cluster-logging-deploy-console_cluster-logging-deploying
QE review:
- [x] QE has approved this change.

Additional information:

Reviewers:

### CODE REVIEW

**Comment by mletalie** [comment]
Hello @kabirbhartiRH,
   Could I please get a QE review on this PR? 
Based on the conversational threads you have been a part of, I believe the conclusion is: 

1) The  "role ARN" text field will be part of the Logging Operator Install UI (when used with ROSA STS cluster).
2) At this point the user does not need to provide the role ARN, but can actually add any info in the text field and they (the user) can continue with installing the operator. 
3) This is live, so we should give the user some instruction/information in the docs; hopefully this text field will either a) be removed as it is not valid for these clusters or b) updated so it works correctly. 

Please let me know if  I am thinking about this correctly. I have not run into this type of situation before. Thanks!  
image

File: modules/cluster-logging-deploy-console.adoc [hunk 1]
```diff
@@ -18,22 +18,27 @@ You can install the {clo} by using the {product-title} web console.
 . In the {product-title} web console, click *Operators* -> *OperatorHub*.
 . Type `OpenShift Logging` in the *Filter by keyword* box.
 . Choose  *Red Hat OpenShift Logging* from the list of available Operators, and click *Install*.
-. Ensure that *A specific namespace on the cluster* is selected under *Installation mode*.
-. Ensure that *Operator recommended namespace* is *openshift-logging* under *Installed Namespace*.
-. Select *Enable operator recommended cluster monitoring on this namespace*.
+ifdef::openshift-rosa[]
+. In the *role ARN* text field, type "no_role_arn". Although the specific ARN for the cluster is not required, the *role ARN* text field must be populated to continue the installation process.
 +
-This option sets the `openshift.io/cluster-monitoring: "true"` label in the `Namespace` object. You must select this option to ensure that cluster monitoring scrapes the `openshift-logging` namespace.
```
Comment (kabirbhartiRH) [inline modules/cluster-logging-deploy-console.adoc:25]: I don't this this stanza should be removed

Comment (mletalie) [reply modules/cluster-logging-deploy-console.adoc:25]: That is correct. The original ticket tied to this PR was to remove step 6 from the current docs b/c (according to the ticket reporter and evident in the UI) that the step was no longer applicable. 
image

**Comment by mletalie** [comment]
> LGTM Asked Logging POC to take a look as well in case of any additional comments.

Hello @kabirbhartiRH,
    Appreciate you taking a look at this issue. I will keep an eye out for any input they provide. Thanks!

**Comment by kabirbhartiRH** [comment]
> > LGTM Asked Logging POC to take a look as well in case of any additional comments.
> 
> Hello @kabirbhartiRH, Appreciate you taking a look at this issue. I will keep an eye out for any input they provide. Thanks!

We can move forward with this! Thanks @mletalie

**Comment by mletalie** [comment]
> > > LGTM Asked Logging POC to take a look as well in case of any additional comments.
> > 
> > 
> > Hello @kabirbhartiRH, Appreciate you taking a look at this issue. I will keep an eye out for any input they provide. Thanks!
> 
> We can move forward with this! Thanks @mletalie
Thanks @kabirbhartiRH !

**Pull request merged**